Index: app/src/main/java/com/example/kiosk/Kiosk_main.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/com/example/kiosk/Kiosk_main.kt b/app/src/main/java/com/example/kiosk/Kiosk_main.kt
--- a/app/src/main/java/com/example/kiosk/Kiosk_main.kt	
+++ b/app/src/main/java/com/example/kiosk/Kiosk_main.kt	
@@ -1,13 +1,18 @@
 package com.example.kiosk.Kiosk_mainKt
 
-import com.example.kiosk.Menu3.Cart
+import com.example.kiosk.Cart
 import com.example.kiosk.Menu1.Menu1
+import com.example.kiosk.Menu2.AbstractRice
+import com.example.kiosk.Menu2.KimchiRice
 import com.example.kiosk.Menu3.Menu3
 import com.example.kiosk.Menu4.Menu4
 import com.example.kiosk.Menu2.RiceMenu
+import com.example.kiosk.Menu2.SelfRice
+import com.example.kiosk.Menu2.ShrimpRice
 
 fun main(){
     var selMeunList = mutableListOf<String>()
+    val menu2List: List<AbstractRice> = listOf(KimchiRice(), SelfRice(), ShrimpRice())
     while(true) {
         println("술집이름")
         println("메뉴를 선택해 주세요.")
@@ -32,7 +37,7 @@
             }
 
             2-> {
-                var f = RiceMenu().riceMenu()
+                var f = RiceMenu().riceMenu1(menu2List)
                 selMeunList.add(f.toString())
             }
 
Index: .idea/misc.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/misc.xml b/.idea/misc.xml
--- a/.idea/misc.xml	
+++ b/.idea/misc.xml	
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
   <component name="ExternalStorageConfigurationManager" enabled="true" />
   <component name="ProjectRootManager" version="2" languageLevel="JDK_17" project-jdk-name="jbr-17" project-jdk-type="JavaSDK">
Index: app/src/main/java/com/example/kiosk/Menu3/Menu3.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/com/example/kiosk/Menu3/Menu3.kt b/app/src/main/java/com/example/kiosk/Menu3/Menu3.kt
--- a/app/src/main/java/com/example/kiosk/Menu3/Menu3.kt	
+++ b/app/src/main/java/com/example/kiosk/Menu3/Menu3.kt	
@@ -1,5 +1,7 @@
 package com.example.kiosk.Menu3
 
+import com.example.kiosk.Cart
+
 data class DryMenu(val name: String, val price: Int, val description: String)
 
 open class Menu3(val cart: Cart) {
Index: .idea/gradle.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/gradle.xml b/.idea/gradle.xml
--- a/.idea/gradle.xml	
+++ b/.idea/gradle.xml	
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
+  <component name="GradleMigrationSettings" migrationVersion="1" />
   <component name="GradleSettings">
     <option name="linkedExternalProjectsSettings">
       <GradleProjectSettings>
Index: app/src/main/java/com/example/kiosk/Menu3/Cart.kt
===================================================================
diff --git a/app/src/main/java/com/example/kiosk/Menu3/Cart.kt b/app/src/main/java/com/example/kiosk/Menu3/Cart.kt
--- a/app/src/main/java/com/example/kiosk/Menu3/Cart.kt	
+++ b/app/src/main/java/com/example/kiosk/Menu3/Cart.kt	
@@ -1,4 +1,7 @@
-package com.example.kiosk.Menu3
+package com.example.kiosk
+
+//import com.example.kiosk.Menu2.RiceMenu
+import com.example.kiosk.Menu3.DryMenu
 
 data class CartItem(val name: String, val price: Int, val quantity: Int)  // 메뉴명, 가격, 수량
 
@@ -13,11 +16,11 @@
         items.add(CartItem(item.name, item.price, quantity))
     }
 
-    /*fun addToCart(item: RiceMenu, quantity: Int) {
-        items.add(CartItem(item.name, item.price, quantity))
-    }
+//    fun addToCart(item: RiceMenu, quantity: Int) {
+//        items.add(CartItem(item.name, item.price, quantity))
+//    }
 
-    fun addToCart(item: DrinkMenu, quantity: Int) {
+    /*fun addToCart(item: DrinkMenu, quantity: Int) {
         items.add(CartItem(item.name, item.price, quantity))
     }*/
 
